src/main/webapp/app/modules/canvas/visualization/visualization-modal/visualization-header.tsx

```diff
@@ -0,0 +1,263 @@
+      x: (props.totalItem * 2) % (3 || 12),
+      height: 3,
+      h: 3,
+      yPosition: 1000,
+      y: 1000,
+      metadataVisual: visualization,
+      views: props.view,
+      datasource: props.view.viewDashboard.dashboardDatasource.id,
+    };
+    newVM = createProperties(newVM);
+    newVM = createFields(newVM);
+    return VisualWrap(newVM);
+  };
+
+  const closeEditDialog = () => {
+    const visual = getVisual();
+    getVisualizationData(visual, props.view);
+  };
+
+  const getTransactionData = async () => {
+    await setTransactionData(props.visual.data);
+    csvLink.current.link.click();
+  };
+
+  useEffect(() => {
+    if (dialog === 'Copy') {
+      const viz = createVisualMetadata(props.visual.metadataVisual);
+      viz.bodyProperties = props.visual.bodyProperties;
+      viz.properties = props.visual.properties;
+      viz.titleProperties = props.visual.titleProperties;
+      viz.fields = props.visual.fields;
+      handleVisualizationClick(viz);
+    } else if (dialog === 'Export') {
+      getTransactionData();
+    }
+  }, [dialog]);
+  return (
+    <>
+      <View backgroundColor="gray-200">
+        <Flex direction="row" justifyContent="space-between" alignContent="center">
+          <Flex direction="column" alignItems="center" justifyContent="space-around">
+            <span>{props.visual.titleProperties.titleText}</span>
+            {props.visual?.data?.length > 0 && (
+              <CSVLink
+                data={transactionData}
+                filename={`${props.visual.titleProperties.titleText}.csv`}
+                className="hidden"
+                ref={csvLink}
+                target="_blank"
+              />
+            )}
+          </Flex>
+          <Flex direction="column" justifyContent="space-around">
+            <MenuTrigger>
+              <ActionButton isQuiet height="size-300">
+                <Settings size={'XS'} aria-label="Default Alert" />
+              </ActionButton>
+
+              {props.isEditMode ? (
+                <Menu onAction={key => setDialog(key)}>
+                  <Item key="Edit" textValue="Edit">
+                    <Edit size="M" />
+                    <Text>
+                      <Translate contentKey="entity.action.edit">Edit</Translate>
+                    </Text>
+                  </Item>
+
+                  <Item key="Copy" textValue="Copy">
+                    <Copy size="M" />
+                    <Text>
+                      <Translate contentKey="entity.action.copy">Copy</Translate>
+                    </Text>
+                  </Item>
+
+                  <Item key="View" textValue="View">
+                    <ViewedMarkAs size="M" />
+                    <Text>
+                      <Translate contentKey="entity.action.view">View</Translate>
+                    </Text>
+                  </Item>
+                  <Item key="Data" textValue="Data">
+                    <Table size="M" />
```

Comment:
use translate function and place the labels in json file

Reply:
already translate is use
key and value are hardcoded